src/services/index.ts

```diff
@@ -0,0 +1,9 @@
+export * from './basic-desc/basic-desc.service';
```

Comment:
@ttsahi means that even if you inherit from the basic class you still need to change the implementation inside ac-layer. so for the current state you can't create a freaky primitive , we need to think about it if we want to add this option in the future.